src/reusable-blocks/class-reusable-blocks.php

```diff
@@ -15,11 +15,7 @@ public function get_ids_from_post( $post_id ) {
 		if ( $post ) {
 			$blocks = \collect( \WPML_Gutenberg_Integration::parse_blocks( $post->post_content ) );
 
-			return $blocks->filter( function( $block ) {
-				return 'core/block' === $block['blockName']
-				       && isset( $block['attrs']['ref'] )
-				       && is_numeric( $block['attrs']['ref'] );
-			})->map( function( $block ) {
+			return $blocks->filter( [ __CLASS__, 'is_reusable' ] )->map( function( $block ) {
```

Comment:
@strategio You may be able to write it like this. I'm not sure if you'll need the `flattern`
```
return $blocks->filter( [ __CLASS__, 'is_reusable' ] )
                       ->pluck( 'attrs' )
                       ->flattern()
                       ->pluck( 'ref' )
                       ->toArray();

```